pkg/controller/atlasmap/action.go

```diff
@@ -46,6 +59,7 @@ func newOperatorActions(log logr.Logger, mgr manager.Manager) []action {
 	return []action{
 		newServiceAction(log.WithValues("type", "service"), mgr),
 		routeAction,
+		consoleLink,
```

Comment:
I think there's a potential nil pointer exception here. If the operator is not running on OpenShift >= 4.3, then `consoleLink` will still be appended to the actions slice as `nil`. When the actions loop runs during reconcile it'll encounter the `nil` action.

Reply:
updated code to handle the nil pointer case